.circleci/config.yml

```diff
@@ -5,8 +5,8 @@ runtest: &runtest
         name: Install GPflow
         command: |
           pip install -U pip
-          pip install --use-feature=2020-resolver -e .
-          pip install --use-feature=2020-resolver -r tests_requirements.txt
```

Comment:
Is that a necessary part of the PR? What is the reasoning for removing it?

Reply:
I could've made a separate PR for that but seemed a bit ridiculous. When looking at the CircleCI output I saw that pip's most recent version is now warning that as of pip 21.0 this will become an error (which would fail all our builds then on).